Nautilus/Handlers/CraftTreeHandler.cs

```diff
@@ -204,4 +205,103 @@ public static void RemoveNode(CraftTree.Type craftTree, params string[] stepsToN
+
+        /// <summary> Steps to the Advanced Materials tab in the Fabricator (<c>Resources/AdvancedMaterials</c>). </summary>
+        public static string[] FabricatorsAdvancedMaterials => new string[] { "Resources", "AdvancedMaterials" };
+
+        /// <summary> Steps to the Electronics tab in the Fabricator (<c>Resources/Electronics</c>). </summary>
+        public static string[] FabricatorsElectronics => new string[] { "Resources", "Electronics" };
+
+        /// <summary> Steps to the Water tab in the Fabricator (<c>Survival/Water</c>). </summary>
+        public static string[] FabricatorWater => new string[] { "Survival", "Water" };
+
+        /// <summary> Steps to the Cooked Food tab in the Fabricator (<c>Survival/CookedFood</c>). </summary>
+        public static string[] FabricatorCookedFood => new string[] { "Survival", "CookedFood" };
+
+        /// <summary> Steps to the Cured Food tab in the Fabricator (<c>Survival/CuredFood</c>). </summary>
+        public static string[] FabricatorCuredFood => new string[] { "Survival", "CuredFood" };
+
+        /// <summary> Steps to the Equipment tab in the Fabricator (<c>Personal/Equipment</c>). </summary>
+        public static string[] FabricatorEquipment => new string[] { "Personal", "Equipment" };
+
+        /// <summary> Steps to the Tools tab in the Fabricator (<c>Personal/Tools</c>). </summary>
+        public static string[] FabricatorTools => new string[] { "Personal", "Tools" };
+
+        /// <summary> Steps to the Deployables tab in the Fabricator (<c>Machines</c>). </summary>
+        public static string[] FabricatorMachines => new string[] { "Machines" };
+
+        /// <summary> Steps to the Vehicles tab in the Mobile Vehicle Bay (<c>Vehicles</c>). </summary>
+        public static string[] ConstructorVehicles => new string[] { "Vehicles" };
+
+        /// <summary> Steps to the Rocket tab in the Mobile Vehicle Bay (<c>Rocket</c>). </summary>
+        public static string[] ConstructorRocket => new string[] { "Rocket" };
+
+        /// <summary> Steps to the Common Modules tab in the Vehicle Upgrade Console (<c>CommonModules</c>). </summary>
+        public static string[] VehicleUpgradesCommonModules => new string[] { "CommonModules" };
+
+        /// <summary> Steps to the Seamoth Modules tab in the Vehicle Upgrade Console (<c>SeamothModules</c>). </summary>
+        public static string[] VehicleUpgradesSeamothModules => new string[] { "SeamothModules" };
+
+        /// <summary> Steps to the Prawn Suit Modules tab in the Vehicle Upgrade Console (<c>ExosuitModules</c>). </summary>
+        public static string[] VehicleUpgradesExosuitModules => new string[] { "ExosuitModules " };
+
+        /// <summary> Steps to the Torpedoes tab in the Vehicle Upgrade Console (<c>Torpedoes</c>). </summary>
+        public static string[] VehicleUpgradesTorpedoes => new string[] { "Torpedoes " };
+#endif
+#if BELOWZERO
```

Comment:
Not sure that you should have a single `#if` for all of them, rather than just putting the if before the specific ones that are exclusive/different between games. Right now you’d have to change two of them if something was changed, it’s better to have them together